client/src/components/SelectedFilters.jsx

```diff
@@ -0,0 +1,95 @@
+import { GrFormClose } from "react-icons/gr";
+
+const SelectedFilters = ({
+  categories,
+  subCategories,
+  setSubCategories,
+  minValue,
+  setMinValue,
+  maxValue,
+  setMaxValue,
+  getProducts,
```

Comment:
Here should be same as PriceRangeSlider, call this onFiltersChange instead of getProducts